### DISCUSSION TITLE
Error while loading PLY file: file not found

### DISCUSSION THREAD

**Discussion by Sapta0802** [NVlabs/sionna#474] [Q&A] [answered]
Hello everyone,

I have created an animation using Blender and exported it in .xml format. I am wondering if it is possible to import this .xml file into Sionna. If so, could you please guide me through the process or point me to any relevant resources?

Any help or advice would be greatly appreciated!

Thank you!

![image](https://github.com/NVlabs/sionna/assets/88283648/61386a42-3e49-4903-b738-b4e05186f4e8)
when I try to do this,   

```
RuntimeError: ​[xml.cpp:1083] Error while loading "C:\Users\Saptarshi Mandal\Desktop\CEWiT\fac_ind_with_human_01.xml" (near line 175, col 3): could not instantiate shape plugin of type "ply": 
[xml.cpp:1083]   [PLYMesh] Error while loading PLY file "Beta_Joints.ply": file not found!
```

**Comment by SebastianCa** [comment]
Hi @Sapta0802,

you need to install the Mitsuba Blender Add-on. For further details, please see [SionnaRT: Scene Creation with Blender](https://www.youtube.com/watch?v=7xHLDxUaQ7c&t=16s).

And please note that animations are currently not supported in SionnaRT.

**Reply by Sapta0802** [reply]
I am perfectly following all the steps that are mentioned in this tutorial.  but still the same problem occurs. I have used the Mitsuba Add-on.

**Reply by SebastianCa** [reply]
Please see comment from @jhoydis below. Did you ensure that you have copied the folder with the `.ply` files?

**Reply by Sapta0802** [reply]
yes, I did that also.

**Comment by pablosreyero** [comment]
Hi there!

I'm not a Sionna developer, but adding to Sebastian's answer (correct me if I'm wrong Sebastian) I think the error has to do with a missing material from your "animation" created in Blender, and Sionna is not recognising that object. You have to take into account that every object in your scene has to have associated a Sionna radio material, and by the name of the object pointed out by the error, i.e., "Beta_Joints.ply" it seems that there's no material associated to it, here you will find the available radio materials from Sionna: [Radio materials](https://nvlabs.github.io/sionna/api/rt.html#radio-materials). For instance, in one of the scenes I created for my simulations, this is how an object of my scene looks like (name wise) "Casino_de_Madrid-itu_metal.ply" (see how at the end of this building's name a Sionna material is highlighted) I'm also assuming that you did not define any custom radio material, but if you did so, then I guess my answer won't apply.

**Comment by jhoydis** [comment]
Hi @Sapta0802,

Could it be that you have only copied the xml file and not the folder with the .ply files?

**Reply by Sapta0802** [reply, marked as answer]
Thank you very much for your help! I discovered that the problem was on my end. I had changed the folder name containing the PLY files, but I forgot to update the path in the XML file. This oversight caused the issue.

**Comment by Chico-Tsunami** [comment]
Hello, a long time ago, I have the same problem, so I think I can help you.
First of all, Jupyter starts in folder --> "C:\Users\ `<YourUser>`\"  so you just need to use load_scene("Desktop/CEWit/fac_ind_with_human_01.xml")

If you are in another location, use this command to check that:
os.getcwd())
You'll just gonna need this library:
import os
